src/interface/src/app/home/share-plan-dialog/share-plan-dialog.component.ts

```diff
@@ -153,8 +153,26 @@ export class SharePlanDialogComponent {
       });
   }
 
+  reloadInvites() {
+    this.invites$ = this.inviteService
+      .getInvites(this.data.planningAreaId)
+      .pipe(tap((_) => (this.isLoading = false)));
+  }
+
   removeAccess(invite: Invite) {
-    this.showSnackbar(`Removed  ${invite.email}`);
+    this.inviteService
+      .deleteInvite(this.data.planningAreaId, invite.id)
+      .subscribe({
+        next: (result) => {
+          this.showSnackbar(`Removed  ${invite.email}`);
+          this.reloadInvites();
```

Comment:
I think related functions were just closing the box, but I thought reloading the invites was a little nicer. Open to changing this.